apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -432,7 +432,29 @@ export class BookingsService_2024_08_13 {
       }
     }
 
-    return formattedBookings;
+    const skip = Math.abs(queryParams?.skip ?? 0);
+    const take = Math.abs(queryParams?.take === 0 ? 1 : queryParams?.take ?? 100);
+    const itemsPerPage = take;
+    const totalPages = Math.ceil(fetchedBookings.totalCount / itemsPerPage);
```

Comment:
I don't understand how this calculation can work - if user passes `queryParams.take = 50` then that is passed to platform library function `getAllUserBookings` which then will return only 50 bookings, right? Because `fetchedBookings` will only be 50 not the total outstanding matching that query.

So then we would just divide 50 / 50